src/gui_common/MouseHoverPanel.cs

```diff
@@ -0,0 +1,197 @@
+﻿using Godot;
+using Godot.Collections;
+
+/// <summary>
+///   The panel that shows what the player is hovering over/inspecting.
+/// </summary>
+public class MouseHoverPanel : PanelContainer
+{
+    [Export]
+    public NodePath CategoriesContainerPath = null!;
+
+    [Export]
+    public NodePath NothingHereContainerPath = null!;
+
+#pragma warning disable CA2213 // Disposable fields should be disposed
+    private Container categoriesContainer = null!;
+    private Container nothingHereContainer = null!;
+#pragma warning restore CA2213 // Disposable fields should be disposed
+
+    private System.Collections.Generic.Dictionary<string, MouseHoverCategory> categories = new();
+    private Array categoriesOrdered = new();
```

Comment:
```suggestion
    private Array categoryControls = new();
```
Would that be a more clear name? From the variable usage I got the impression that this is used for that. The current name `categoriesOrdered` sounds grammatically incorrect so it needs a new name.